src/net.h

```diff
@@ -145,6 +145,8 @@ class CConnman
         std::vector<std::string> vSeedNodes;
         std::vector<CSubNet> vWhitelistedRange;
         std::vector<CService> vBinds, vWhiteBinds;
+        bool m_use_addrman_outgoing = true;
```

Comment:
Done. Note that now, the default options are to not make any outbound connections (I put it to `true` to have it make outbound connections by default, like bitcoind does).